src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -252,7 +252,7 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& function) const {
     manager.register_pass<pass::GRUBlockCellReplacer>();
 
     // TODO: reimplement TransposeSinking that does not corrupt filters for Convolution
-    manager.register_pass<ov::frontend::tensorflow::pass::TransposeSinking>();
+    // manager.register_pass<ov::frontend::tensorflow::pass::TransposeSinking>();
```

Comment:
Did you miss a changing copyrights of this file in case it is changed?

Reply:
Done